src/ReactNativeZoomableView.tsx

```diff
@@ -589,11 +640,20 @@ class ReactNativeZoomableView extends Component<
 
     const gestureCenterPoint = calcGestureCenterPoint(e, gestureState);
 
-    const zoomCenter = {
+    let zoomCenter = {
       x: gestureCenterPoint.x - this.state.originalPageX,
       y: gestureCenterPoint.y - this.state.originalPageY,
     };
 
+    if (this.props.staticPinPosition) {
```

Comment:
I think we can put the `calcGestureCenterPoint` call in an `else` block

Reply:
Unfortunately we need `gestureCenterPoint` further down on line 701